body.tex

```diff
@@ -490,35 +597,10 @@ \subsubsection{Completing the Work}
 \item (If applicable, i.e. it is not a bucket, as defined in \S\ref{sec:bucket}) its concrete deliverable has been achieved.
 \end{enumerate}
 
+``Bucket'' epics should be closed at the end of the time box (i.e. end of half fiscal year and end of fiscal year).
```

Comment:
We may also assign buckets to a specific milestones and close them when the milestones is achieved e.g. DR1 Bucket Epic or  DP0.2 Bucket epic.